ambari-server/src/main/resources/stacks/HDP/2.5/services/HIVE/kerberos.json

```diff
@@ -150,7 +150,7 @@
             {
               "webhcat-site": {
                 "templeton.kerberos.secret": "secret",
-                "templeton.hive.properties": "hive.metastore.local=false,hive.metastore.uris=${clusterHostInfo/hive_metastore_host|each(thrift://%s:9083, \\\\,, \\s*\\,\\s*)},hive.metastore.sasl.enabled=true,hive.metastore.execute.setugi=true,hive.metastore.warehouse.dir=/apps/hive/warehouse,hive.exec.mode.local.auto=false,hive.metastore.kerberos.principal=hive/_HOST@${realm}"
+                "templeton.hive.properties": "hive.metastore.local=false,hive.metastore.uris=${clusterHostInfo/hive_metastore_hosts|each(thrift://%s:9083, \\\\,, \\s*\\,\\s*)},hive.metastore.sasl.enabled=true,hive.metastore.execute.setugi=true,hive.metastore.warehouse.dir=/apps/hive/warehouse,hive.exec.mode.local.auto=false,hive.metastore.kerberos.principal=hive/_HOST@${realm}"
```

Comment:
This value will need to be updated in the user-suppled Kerberos descriptor when upgrading to Ambari 2.7.  This needs to be done in the UpgradeCatalog270 class. See `org.apache.ambari.server.upgrade.UpgradeCatalog270#updateKerberosDescriptorArtifact`.

Reply:
Done